ring_doorbell/__init__.py

```diff
@@ -44,7 +45,6 @@ def __init__(self, username, password,
         self.params = None
         self._persist_token = persist_token
         self._push_token_notify_url = push_token_notify_url
-        self._timeout = timeout
```

Comment:
Please, restore this attribute as this is being used as a timeout for the requests (see https://github.com/tchellomello/python-ring-doorbell/pull/167/files#diff-98d24069e9865c04cb18bcad289e3881L139 for example)

Reply:
Apologies, I made the original changes on the stable branch, I missed that one :)